client/src/components/Rating.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+// Styling imports
+import { RatingWrapper, OverviewWrapper, OverviewText, CategoryWrapper, RatingContainer} from './styles/staticStyles.style'
+import { TextContainer, RatingText } from './styles/generalUse.style';
+import { Bar } from './styles/propStyles.style';
+import { FaStar } from "react-icons/fa";
+
+const Rating = (props) => {
+  const { cleanAvg, commAvg, accuracyAvg, valueAvg, locationAvg, checkinAvg } = props.overview;
+  const cleanPercent = Math.floor((Number(cleanAvg) / 5) * 100);
```

Comment:
It'd be cleaner to put the `Math.floor((Number(n) / 5) * 100);` stuff into a function since it's repeated so many times.